src/app/settings/settings.component.html

```diff
@@ -54,11 +54,10 @@
 
       <div class="fs-15px pt-30px">BitClout Theme</div>
       <br />
-      <select name="theme" id="theme" (change)="selectChangeHandler($event)" class="form-control selector">
-        <option *ngIf="whichTheme == 'dark'" value="dark">Dark Theme</option>
-        <option *ngIf="whichTheme == 'light'" value="light">Light Theme</option>
-        <option *ngIf="whichTheme == 'light'" value="dark">Dark Theme</option>
-        <option *ngIf="whichTheme == 'dark'" value="light">Light Theme</option>
+      <select [value]="whichTheme" name="theme" id="theme" (change)="selectChangeHandler($event)" class="form-control selector">
+        <option value="dark">Dark Theme</option>
+        <option value="light">Light Theme</option>
+        <option value="iceydark">Icey Dark Theme</option>
```

Comment:
Please correct spelling to `icy`

Reply:
as much as I love Gucci Mane, I don't think most users will understand that it's an intentional typo.

Reply:
haha okay okay fair enough 💯 lolol